CHANGELOG.md

```diff
@@ -1,335 +1,348 @@
+| Implemented adding an item to a Wish List from the category page                                           | [#3105][] |
+| Fixed a Wish List bug that prevented you from removing a product if the Wish List had 2 or more products   | [#3121][] |
+| Implemented adding an item to a Wish List from the cart                                                    | [#3130][] |
+| Implemented adding an item to the cart from a Wish List                                                    | [#3170][] |
+| Created a re-usable hook for Wish List logic for re-use in various components                              | [#3182][] |
+| Fixed a Wish List bug that allowed users to collapse the Wish List section when it only had a single entry | [#3184][] |
+| Implemented adding an item from the cart to multiple wishlists                                             | [#3207][] |
+| Added message to display when the Wish List is empty                                                       | [#3228][] |
+| Fixed a Wish List bug in the Create Wish List dialog that prevented users from creating a new Wish List    | [#3242][] |
+| Cleaned up Multi Wish List code                                                                            | [#3246][] |
+| Created a single Wish List button component to use throughout the application                              | [#3249][] |
 
-The Buildpack scaffolding tool for creating new projects now lets you specify a custom template and version.
-This feature lets you create a storefront project based on a non-Venia template or use a pre-release version of the Venia template.
+[#3242]: https://github.com/magento/pwa-studio/pull/3242
+[#3184]: https://github.com/magento/pwa-studio/pull/3184
+[#3249]: https://github.com/magento/pwa-studio/pull/3249
+[#3228]: https://github.com/magento/pwa-studio/pull/3228
+[#3182]: https://github.com/magento/pwa-studio/pull/3182
+[#3121]: https://github.com/magento/pwa-studio/pull/3121
+[#3207]: https://github.com/magento/pwa-studio/pull/3207
+[#3170]: https://github.com/magento/pwa-studio/pull/3170
+[#3130]: https://github.com/magento/pwa-studio/pull/3130
+[#3105]: https://github.com/magento/pwa-studio/pull/3105
+[#3048]: https://github.com/magento/pwa-studio/pull/3048
+[#3041]: https://github.com/magento/pwa-studio/pull/3041
+[#3049]: https://github.com/magento/pwa-studio/pull/3049
+[#3246]: https://github.com/magento/pwa-studio/pull/3246
+### Cypress tests
 
-![Custom scaffolding template](https://user-images.githubusercontent.com/4692281/112341120-25f5a680-8c8f-11eb-9174-122dcc1eb5c4.png)
+[Cypress](https://www.cypress.io/) is an end-to-end testing suite written in JavaScript.
+This release adds this framework to the PWA Studio project to increase testing automation and reduce the time spent on manual testing.
+This will enable the team to release new versions faster and more often.
 
-### MegaMenu component
+Integration tests for PageBuilder and the new Wish List feature are included in this release.
 
```

Comment:
Could we add a link to where these tests live within the PWA Studio repository?

We would want our customers to be able to utilize these tests in their own CI/CD pipelines eventually, so might be worth adding a note around that.